src/ansys/meshing/prime/internals/launcher.py

```diff
@@ -226,8 +226,8 @@ def launch_prime(
     if ip == defaults.ip():
         port = utils.get_available_local_port(port)
 
-    if 'PYPRIMEMESH_LAUNCH_CONTAINER' in os.environ:
-        container_name = 'ansys-prime-server'
+    if bool(int(os.environ.get('PYPRIMEMESH_LAUNCH_CONTAINER', '0'))):
```

Comment:
Just to improve readability a bit.
```suggestion
    launch_container = bool(int(os.environ.get('PYPRIMEMESH_LAUNCH_CONTAINER', '0')))
    if launch_container:
```